go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -501,14 +535,30 @@ func getPodInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
 		podInfo.OfportNum = ovsInterfaceInformation.Ofport
 	}
 
-	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, cmd, podInfo.NodeName)
+	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, podInfo.SbCommand, podInfo.NodeName)
 	if err != nil {
 		return nil, err
 	}
 
 	return podInfo, err
 }
 
+func getSwitchPortMacAddress(coreclient *corev1client.CoreV1Client, restconfig *rest.Config, podInfo *PodInfo, ovnNamespace, portPrefix string) (string, error) {
```

Comment:
Nit: The function name is misleading (a bit).  We're actually going to dump port bindings that correspond to router ports.  I'd call it `getRouterPortMacAddress()` or, maybe better, `getLogicalPortMacAddress()`.

Reply:
changed the function into `getRouterPortMacAddress()`.